doc/development.md

```diff
@@ -39,23 +39,23 @@ should display something like `/home/<userdir>/.gvm/pkgsets/go1.11/global`
 Clone operator repo:
 ```
 cd $GOPATH
-go get github.com:pravega/pravega-operator
+go get github.com/pravega/pravega-operator
 ```
 This should clone operator code under `$GOPATH/src/github.com/pravega/pravega-operator`
 
-### Install dep
-```
-curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-```
-This may prompt for creation of `GOBIN` directory. Please create it and then issue this command again.
+Set the following system property to use go modules when the version of go is below 1.13 and the operator code is cloned inside the GOPATH
```

Comment:
This text can be removed since our minimum version is 1.13+
```
Set the following system property to use go modules when the version of go is below 1.13 and the operator code is cloned inside the GOPATH
```	
curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh	`GO111MODULE=on`
```

Reply:
Done